scripts/generate-docs.js

```diff
@@ -0,0 +1,46 @@
+
+const path = require('path');
```

Comment:
I think we can probably use some stuff in `readme-api-generator`.

Reply:
While I didn't find any stuff is useful for our use case...

Reply:
OK. We should discuss, since part of the reason I wrote it was to make it easier to generate docs.